<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1381</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0);</string>
    </property>
    <widget class="QLabel" name="welcomeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>671</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255);
font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(61, 187, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amazon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Best Buy</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="storeLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Choose A Store</string>
     </property>
    </widget>
    <widget class="QLabel" name="ItemLable">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Item Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="item">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>290</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(61, 187, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>400</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(61, 187, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>30</y>
       <width>651</width>
       <height>721</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(61, 187, 255);</string>
     </property>
     <column>
      <property name="text">
       <string>Item Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item Price</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="howToUse">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>680</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(61, 187, 255);
border-radius: 40px;
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>How To Use</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
